src/components/navigation/NavItem.js

```diff
@@ -30,16 +30,36 @@ export const NavItem = props => {
     <a href={path} className={classes}>
       <div className="border">
         <div className="icon">
-          <Icon>{logo}</Icon>
+          {brandLogo ? (
+            <img src={brandLogo} className="brand-logo" alt={label} />
+          ) : (
+            <Icon>{logo}</Icon>
+          )}
         </div>
       </div>
       <div className="label">{label}</div>
     </a>
   );
 };
 
+const iconImagePropsCheck = (props, propName, componentName) => {
+  if (!props.logo && !props.brandLogo) {
```

Comment:
This removes the type check that requires them to be strings

Reply:
fixed